class.jetpack.php

```diff
@@ -553,6 +553,7 @@ private function __construct() {
 		} );
 
 		$this->connection_manager = new Connection_Manager( );
+		Jetpack_Sync_Main::init()->set_connection_manager( $this->connection_manager );
```

Comment:
Why do we care so much if there's more than once instance of the connection manager? Having getters hides type information and introduces confusing indirection. I don't understand why we can't just construct this class when we need it.